src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,62 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int ARRAY_MAX_SIZE = 10;
+    private Pair<K, V>[] storage;
+    private int indexOfNextPair = 0;
```

Comment:
This field will be 0 by default, also better to rename to size
```suggestion
    private int indexOfNextPair;
```